__tests__/hooks/useRenderMarkdownAsHTML.js

```diff
@@ -8,21 +8,21 @@ jest.setTimeout(timeouts.test);
 test('renderMarkdown should use Markdown-It if not set in props', async () => {
   const { pageObjects } = await setupWebDriver();
 
-  await expect(pageObjects.runHook('useRenderMarkdownAsHTML', [], fn => fn('Hello, World!'))).resolves.toBe(
-    '<p>Hello, World!</p>\n'
+  await expect(pageObjects.runHook('useRenderMarkdownAsHTML', [], fn => fn('Hello, World!'))).resolves.toContain(
+    '<p>Hello, World!</p>'
   );
 });
 
 test('renderMarkdown should use custom Markdown transform function from props', async () => {
   const { pageObjects } = await setupWebDriver({
     props: {
-      renderMarkdown: text => text.toUpperCase()
+      renderMarkdown: text => `<p>${text.toUpperCase()}</p>`
```

Comment:
We no longer support Markdown engine outputting text without any HTML tags.

Markdown engine should emit at least a `<p>`.